modules/kmm-installing-using-web-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-install-using-web-console_{context}"]
+
+= Installing the Kernel Module Management Operator using the web console
+
+As a cluster administrator, you can install the Kernel Module Management (KMM) Operator using the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Install the Kernel Module Management Operator:
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose *Kernel Module Management Operator* from the list of available Operators, and then click *Install*.
+
+.. On the *Install Operator* page, select *a specific namespace on the cluster*, select the `openshift-kmm` namespace, and then click *Install*.
+
+.Verification
+
+To verify that the Special Resource Operator installed successfully:
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+. Ensure that *Kernel Module Management Operator* is listed in the *openshift-operators* project with a *Status* of *InstallSucceeded*.
```

Comment:
The suggested Namespace is `openshift-kmm` . I think we should replace that one with 'openshift-kmm' or the user entered namespace.

Reply:
ok. corrected.